kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -250,6 +290,14 @@ public ClassLoader getClassLoader() {
     }
 
     public XStream getXstream() {
+        // if someone is invoking this method is very likely that xtream will change
+        // so a new one will need to be created
+        ignoreLock.lock();
```

Comment:
xstream not mutable, for completeness we need to reinstantiate xstreamignore if needed